### DISCUSSION TITLE
Project questions: title and card

### DISCUSSION THREAD

**Discussion by liangchow** [alshedivat/al-folio#1810] [Q&A] [answered]
Hi, I have two questions:

- How do I capitalize the first letter of project title on cards on the "Projects" page? For example, in "_projects/1_project.md," the title is "Project 001," but on the "Projects" page, the card title shows "project 001." I can't any text-related line on "_sass/_base.scss". Am I missing something?

- I really like this user does his/her project cards, i.e., hovering to show project titles. It appears the user uses an older al-folio version. I tried modifying css, pages, and project, but it doesn't work. Can anyone help?

Thank you!

**Comment by george-gca** [comment, marked as answer]
1. Yes, this is currently set in the [_includes/projects.html](https://github.com/alshedivat/al-folio/blob/a21bd492bc9950f86f160baea1a2e31d3cc6227d/_includes/projects.html#L16) and in the [_includes/projects_horizontal.html](https://github.com/alshedivat/al-folio/blob/master/_includes/projects_horizontal.html#L18), depending which do you use, more specifically with the class `text-lowercase`. Remove this class and it will work as you expect it.
2. No idea how he does this, but did you try to check if the source code for his site is available? If so, you could find some clues by digging in the code, or maybe looking at some of the commits' titles.
